examples/tokenvm/orderbook/orderbook.go

```diff
@@ -86,10 +86,10 @@ func (o *OrderBook) Add(txID ids.ID, actor codec.Address, action *actions.Create
 		return
 	case !ok && o.trackAll:
 		o.c.Logger().Info("tracking order book", zap.String("pair", pair))
-		h = heap.New[*Order, float64](o.maxOrdersPerPair+1, true)
+		h = heap.New[ids.ID, *Order, float64](o.maxOrdersPerPair+1, true)
 		o.orders[pair] = h
 	}
-	h.Push(&heap.Entry[*Order, float64]{
+	h.Push(&heap.Entry[ids.ID, *Order, float64]{
 		ID:    order.ID,
```

Comment:
I don't know if there is a way around this, but this is not very golang-y imo (to have all three of these values populated by `order` derivatives)